apisix/plugins/limit-conn.lua

```diff
@@ -33,6 +33,7 @@ local schema = {
         rejected_code = {
             type = "integer", minimum = 200, maximum = 599, default = 503
         },
+        degradation = {type = "boolean", default = false}
```

Comment:
Would be better to use "allow_degradation"?

Reply:
done